src/http_microhttpd.c

```diff
@@ -498,7 +498,7 @@ static int authenticated(struct MHD_Connection *connection,
 
 	if (check_authdir_match(url, config->denydir)) {
 		auth_client_deauth(client->id, "client_deauth");
-		snprintf(redirect_to_us, 128, "http://%s:%u/", config->gw_address, config->gw_port);
+		snprintf(redirect_to_us, 128, "http://%s/", config->gw_address);
```

Comment:
`128` could be replaced with `sizeof(redirect_to_us)`. But it is not important for this PR.